### PR TITLE
Fixes weather, robots

### PR DESCRIPTION
**Pull request by realbuffalowing**
Lightning no longer strikes you indoors. Thunder is now in the ambiance channel. Borgs no longer have to pray for a roboticist to be online to not die in the middle of an RP.

### CODE REVIEW

File: code/modules/admin/verbs/lightning_strike.dm [hunk 1]
```diff
@@ -65,7 +65,9 @@
 	var/sound = get_sfx("thunder")
 	for(var/mob/M in player_list)
 		if((P && M.z in P.expected_z_levels) || M.z == T.z)
-			M.playsound_local(get_turf(M), soundin = sound, vol = 70, vary = FALSE, is_global = TRUE)
+			if(!(M && M.is_preference_enabled(/datum/client_preference/play_ambiance)))	return
```
Comment (nik707) [inline code/modules/admin/verbs/lightning_strike.dm]: is it really supposed to be fucking "ambiance"

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -51,7 +51,7 @@
 	var/obj/item/device/radio/borg/radio = null
 	var/obj/item/device/communicator/integrated/communicator = null
 	var/mob/living/silicon/ai/connected_ai = null
-	var/obj/item/weapon/cell/cell = null
+	var/obj/item/weapon/cell/super/cell = null
```
Comment (nik707) [inline code/modules/mob/living/silicon/robot/robot.dm]: HiCap, not super

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -137,9 +137,9 @@
 		C.wrapped = new C.external_type
 
 	if(!cell)
-		cell = new /obj/item/weapon/cell(src)
-		cell.maxcharge = 7500
-		cell.charge = 7500
+		cell = new /obj/item/weapon/cell/super(src)
```
Comment (nik707) [inline code/modules/mob/living/silicon/robot/robot.dm]: see above

File: code/modules/admin/verbs/lightning_strike.dm (hunk 1 above)
Comment (realbuffalowing) [reply code/modules/admin/verbs/lightning_strike.dm]: cannot make this up, its spelled 2 different ways in the code. this is the correct way for functionality.

**Pull request merged**